python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -316,13 +252,87 @@ def _computePsfStats(self, sources):
         psfStarDeltaSizeScatter = sigmaMad(starSize - psfSize, scale='normal')
         psfStarScaledDeltaSizeScatter = psfStarDeltaSizeScatter/starSizeMedian**2.
 
-        psfStats.nPsfStar = int(nPsfStar)
-        psfStats.psfStarDeltaE1Median = float(psfStarDeltaE1Median)
-        psfStats.psfStarDeltaE2Median = float(psfStarDeltaE2Median)
-        psfStats.psfStarDeltaE1Scatter = float(psfStarDeltaE1Scatter)
-        psfStats.psfStarDeltaE2Scatter = float(psfStarDeltaE2Scatter)
-        psfStats.psfStarDeltaSizeMedian = float(psfStarDeltaSizeMedian)
-        psfStats.psfStarDeltaSizeScatter = float(psfStarDeltaSizeScatter)
-        psfStats.psfStarScaledDeltaSizeScatter = float(psfStarScaledDeltaSizeScatter)
+        summary.nPsfStar = int(nPsfStar)
+        summary.psfStarDeltaE1Median = float(psfStarDeltaE1Median)
+        summary.psfStarDeltaE2Median = float(psfStarDeltaE2Median)
+        summary.psfStarDeltaE1Scatter = float(psfStarDeltaE1Scatter)
+        summary.psfStarDeltaE2Scatter = float(psfStarDeltaE2Scatter)
+        summary.psfStarDeltaSizeMedian = float(psfStarDeltaSizeMedian)
+        summary.psfStarDeltaSizeScatter = float(psfStarDeltaSizeScatter)
+        summary.psfStarScaledDeltaSizeScatter = float(psfStarScaledDeltaSizeScatter)
 
-        return psfStats
+    def update_wcs(self, summary, wcs, bbox, visitInfo):
```

Comment:
Ditto on adding another descriptor to the function name (e.g. `update_wcs_stats`)?